pkg/http/server.go

```diff
@@ -257,7 +257,7 @@ func (s *server) useMaxBodySize(r chi.Router) {
 	}
 
 	maxSize := int64(s.config.MaxRequestBodySizeMB) << 20 // To bytes
-	log.Infof("Enabled max body size HTTP middleware with size %d MB", maxSize)
+	log.Infof("Enabled max body size HTTP middleware with size %d bytes", maxSize)
```

Comment:
Please print the size in MB for logging. Bytes is essentially useless to humans without getting the calculator out at this order of magnitude.

Reply:
Yeah that's a good callout. Feel free to make a suggestion too!
```suggestion
	log.Infof("Enabled max body size HTTP middleware with size %d MB", s.config.MaxRequestBodySizeMB)
```